packages/gatsby-source-lever/src/gatsby-node.js

```diff
@@ -3,6 +3,50 @@ const normalize = require(`./normalize`)
 
 const typePrefix = `lever__`
 
+exports.createSchemaCustomization = async ({ actions }) => {
+  const typeDefs = `
+    type lever implements Node {
+      additional: String
+      additionalPlain: String
+      applyUrl: String
+      createdAt: Date
+      description: String
+      descriptionPlain: String
+      id: ID!
+      hostedUrl: String
+      lever_id: String
+      text: String
+      categories: LeverCategories
+      lists: [LeverList]
+      internal: LeverInternal!
+    }
+
+    type LeverCategories {
+      commitment: String
+      level: String
+      location:String
+      team: string
```

Comment:
Should be uppercase `String`:

```suggestion
      team: String
```